[2024-01-29T18:10:25.337+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Dag_ses_burnoye_forecast.weather_to_bd scheduled__2024-01-28T09:30:00+00:00 [queued]>
[2024-01-29T18:10:25.349+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Dag_ses_burnoye_forecast.weather_to_bd scheduled__2024-01-28T09:30:00+00:00 [queued]>
[2024-01-29T18:10:25.350+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 10
[2024-01-29T18:10:25.366+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): weather_to_bd> on 2024-01-28 09:30:00+00:00
[2024-01-29T18:10:25.372+0000] {standard_task_runner.py:57} INFO - Started process 186 to run task
[2024-01-29T18:10:25.375+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Dag_ses_burnoye_forecast', 'weather_to_bd', 'scheduled__2024-01-28T09:30:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/dag_ses_burnoye.py', '--cfg-path', '/tmp/tmpofz9vm49']
[2024-01-29T18:10:25.378+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask weather_to_bd
[2024-01-29T18:10:25.429+0000] {task_command.py:416} INFO - Running <TaskInstance: Dag_ses_burnoye_forecast.weather_to_bd scheduled__2024-01-28T09:30:00+00:00 [running]> on host 88cd4f1bae89
[2024-01-29T18:10:25.517+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Dag_ses_burnoye_forecast' AIRFLOW_CTX_TASK_ID='weather_to_bd' AIRFLOW_CTX_EXECUTION_DATE='2024-01-28T09:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-28T09:30:00+00:00'
[2024-01-29T18:10:26.946+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-29T18:10:26.961+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Dag_ses_burnoye_forecast, task_id=weather_to_bd, execution_date=20240128T093000, start_date=20240129T181025, end_date=20240129T181026
[2024-01-29T18:10:26.993+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-29T18:10:27.024+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
